src/styles/_reset.scss

```diff
@@ -18,47 +20,47 @@ body {
     display: inline-block;
   }
 
-	@include line-styles {
-		display: block;
-	}
+  @include line-styles {
+    display: block;
+  }
 }
 
 html {
-	&.has-modal {
-		&,
-		& body {
-			overflow: hidden;
-		}
-	}
+  &.has-modal {
+    &,
+    & body {
+      overflow: hidden;
+    }
+  }
 }
 
 a {
   color: $c-current;
   text-decoration: none;
-	&:hover {
-		text-decoration: none;
-	}
+  &:hover {
+    text-decoration: none;
+  }
 
-	&:focus {
-		text-decoration: none;
+  &:focus {
+    text-decoration: none;
 
-		outline: none;
-	}
+    outline: none;
+  }
 }
 
 li:focus {
-	outline: none;
+  outline: none;
 }
 
 img {
-	display: inline-block;
+  display: inline-block;
```

Comment:
inline-block will always have empty space below the image like other inline/inline-block elements

Reply:
It's yours reset file :P